app/components/utils/system-alert/component.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component';
+
+export default class extends Component {
+  get type () {
+    const t = this.args.type;
+    if (t === 'Waarschuwing') {
+      return 'warning';
+    } else if (t === 'Dringend') {
+      return 'error';
+    } else {
+      return null;
+    }
+  }
+}
```

Comment:
I cannot quite see where this new component is used from? What's the plan here?

Reply:
It gets used [here](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/287/files#diff-f63965ba48df687541db46e0e1b2c2ecR5-R9). It's not much more than a wrapper to abstract away the type that the alert uses (map it to the type of the standard alert component)

Reply:
If you see a cleaner way of doing this, I'm all ears :ear: :wink:

Reply:
I'm not sure, but you linked to this PR's changes? I just checked out your branch and globally searched for `system-alert` to find out where the component is used. The searched yielded no results. This is why I asked a question about it.

Reply:
mm ... the html-anchor doesn't seem to work ... click the link and search for `Utils::SystemAlert` :-)

Reply:
Ohhhh, of course, the new angle bracket syntax! OK no, this is fine! My bad!